$NetBSD: patch-ad,v 1.4 2010/11/06 15:05:50 ahp-nils Exp $

Compatibility patch for systems not using alloca.h and some
other things. Patch commited upstream.

--- logrotate.c.orig	2010-06-28 08:04:56.000000000 +0000
+++ logrotate.c
@@ -1,5 +1,9 @@
 #include <sys/queue.h>
+#ifdef _ALLOCA_H
 #include <alloca.h>
+#else
+#include <limits.h>
+#endif
 #include <ctype.h>
 #include <dirent.h>
 #include <errno.h>
